README.ko.md

```
@@ -0,0 +1,218 @@
+
+## not complete
+
+[English](README.md) | **한국어**
+
+---
+
+### 설치
+
+cargo를 사용한다.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)